demo/play-demo.sh

```diff
@@ -0,0 +1,207 @@
+sleep 2
+exec oc login https://159.69.209.188:8443 --token=Hz8JKkb80xa7rptVGJhq6aerZUGwyaRVYJ7zBQfFly8
+sleep 3
+
+clear && sleep 1
+echo "# Create a new project"|pv -qL 10
+sleep 1
+exec oc new-project demo
+sleep 5
+
+clear && sleep 1
+echo "# Create a directory named demo and subsequently enter in"|pv -qL 10
+sleep 2
+exec mkdir demo
+sleep 1
+exec cd demo
+sleep 5
+
+clear && sleep 1
+echo "# Create a pom.xml with the following content"|pv -qL 10
+sleep 2
+echo "<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>me.fruitsand</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Spring Boot - Demo</name>
+    <description>Spring Boot - Demo</description>
+    <packaging>pom</packaging>
+    <modules>
+        <module>fruit-backend-sb</module>
+        <module>fruit-client-sb</module>
+    </modules>
+</project>"
+sleep 10
+cat > pom.xml << ENDOFFILE
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>me.fruitsand</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Spring Boot - Demo</name>
+    <description>Spring Boot - Demo</description>
+    <packaging>pom</packaging>
+    <modules>
+        <module>fruit-backend-sb</module>
+        <module>fruit-client-sb</module>
+    </modules>
+</project>
+ENDOFFILE
+clear && sleep 1
+echo "# Create a new client project using the REST HTTP client template proposed by the scaffolding tool"|pv -qL 10
+sleep 3
+exec hal component spring-boot \
+   -i fruit-client-sb \
+   -g me.fruitstand \
+   -p me.fruitstand.demo \
+   -s 2.1.6.RELEASE \
+   -t client \
+   -v 1.0.0-SNAPSHOT \
+   --supported=false  \
+   fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Create a backend project interactively and use as template the crud type and fruit-backend-sb as maven project name"|pv -qL 10
+sleep 3
+exec hal component spring-boot fruit-backend-sb
+sleep 3
+
+clear && sleep 1
+echo "# Build the projects"|pv -qL 10
+echo "# Compile and generate the uber jar of the Spring Boot application client"|pv -qL 10
+sleep 3
+exec mvn package -f fruit-client-sb
+sleep 3
+
+clear && sleep 1
+echo "# Repeat the command executed previously for the CRUD - backend microservice."|pv -qL 10
+echo "# We need to use the kubernetes profile because the project is set up to work both locally using H2 database for quick testing and "remotely" using a PostgreSQL database."|pv -qL 10
```

Comment:
are the quoted on `remotely` wrong perhaps?

Reply:
yes, fixed